mobile/src/main/java/rs/readahead/washington/mobile/data/database/WashingtonSQLiteOpenHelper.java

```diff
@@ -94,7 +94,7 @@ public void onCreate(SQLiteDatabase db) {
 
     @Override
     public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
-        switch (oldVersion) {
+        switch (newVersion) {
```

Comment:
Regarding this... Here, in case of a new version, the database will be modified according to the latest version only.

In case the user has not updated the application for a long time, it is possible that some changes will be skipped (versions in the meantime). If we update referring to the old (existing) version, all changes from the current to the last version will be made.